BookingsApi/BookingsApi.DAL/Migrations/20231102114115_AddWorkPhoneToJudicialPersonStaging.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace BookingsApi.DAL.Migrations
+{
+    public partial class AddWorkPhoneToJudicialPersonStaging : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
+                name: "WorkPhone",
+                table: "JudiciaryPersonsStaging",
+                type: "nvarchar(max)",
```

Comment:
does a workphone number column need to be varchar (max)? I think the migration generated defaults to that maybe?

Reply:
they all have nvarchar max oddly. I can't get information about the max length for a lot of these properties so I've kept it in line with what's present. There's definitely room for optimisation but the difference is not noticeable for the one column

Reply:
Yeah think the properties need to be annoted with a string length attribute. Probably not worth doing if it means retroactively doing it to all columns